src/configuration/prima.rs

```diff
@@ -77,13 +82,45 @@ impl FromStr for Environment {
     }
 }
 
-impl ToString for Environment {
-    fn to_string(&self) -> String {
+impl Display for Environment {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         match self {
-            Environment::Dev => "dev".to_string(),
-            Environment::Qa => "qa".to_string(),
-            Environment::Staging => "staging".to_string(),
-            Environment::Production => "production".to_string(),
+            Environment::Dev => write!(f, "dev"),
+            Environment::Qa => write!(f, "qa"),
+            Environment::Staging => write!(f, "staging"),
+            Environment::Production => write!(f, "production"),
+        }
+    }
+}
+
+/// Represents the country in which the datadog client runs.
+/// This is useful for enforcing rules based on country for every application that uses the library.
+#[derive(PartialEq, Debug, Clone)]
+pub enum Country {
+    It,
+    Es,
+    Uk,
+}
+
+impl FromStr for Country {
+    type Err = PrimaDatadogError;
+
+    fn from_str(s: &str) -> Result<Self, Self::Err> {
+        match s {
+            "it" => Ok(Self::It),
+            "es" => Ok(Self::Es),
+            "uk" => Ok(Self::Uk),
+            _ => Err(PrimaDatadogError::WrongCountryDefinition),
+        }
+    }
+}
+
+impl Display for Country {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            Country::It => write!(f, "it"),
+            Country::Es => write!(f, "es"),
+            Country::Uk => write!(f, "uk"),
```

Comment:
Could this be done with `strum`?

Reply:
it could but I'm not sure it's worth adding a dep for? E.g. `Environment` does the same thing

Reply:
I was thinking about that too, let's leave it like this. It's not like we are handling hundreds of enums or whatever